e2e/api/steps.js

```diff
@@ -408,7 +408,7 @@ const assertSubmittedEvent = async (expectedState, submittedCallbackResponseCont
 
 const assertContainsPopulatedFields = returnedCaseData => {
   for (let populatedCaseField of Object.keys(caseData)) {
-    assert.equal(populatedCaseField in returnedCaseData, true,
+    chai.assert.include(populatedCaseField, returnedCaseData,
```

Comment:
Can we please remove 3rd argument from this method? The built-in message should be more descriptive.